src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -88,6 +92,67 @@ const freeformFilters: FilterDefinition<
       };
     },
   },
+  {
+    keywords: 'contains',
+    description: tl('LoadoutFilter.Contains'),
+    format: 'freeform',
+    suggestionsGenerator: ({ d2Definitions, loadouts, selectedLoadoutsStore }) => {
+      if (!d2Definitions || !loadouts) {
+        return [];
+      }
+
+      return loadouts.flatMap((loadout) => {
+        if (loadout.classType !== selectedLoadoutsStore?.classType) {
```

Comment:
We have a helper for "class type matches store" that correctly handles "any class" loadouts.